util/texlive/Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM debian:jessie
+MAINTAINER nate.brennand@clever.com
+# Install latex related tooling:
+RUN apt-get update -y && \
+    apt-get install ca-certificates -y && \
```

Comment:
Good to know about the `jessie-curl`, but I think I'd rather just make this one "standard" with `debian:jessie` since we use that almost everywhere right now.